source/features/sticky-sidebar.css

```diff
@@ -35,14 +35,6 @@ Exclusively use simple sums and use `0px` instead of `0`
 .rgh-sticky-sidebar {
 	position: sticky;
 	top: var(--rgh-sticky-sidebar-offset, 0) !important;
-
-	/* `sticky` will trap the `fixed` dialog’s z-index in some sidebars #3732 */
-	/* Higher than sticky readme header #4192 */
-	z-index: 90;
```

Comment:
Typical bugfix causing a billion other bugs.

Deleted the bugfix, none of the other issues are present anymore